tools/doc/package.json

```diff
@@ -7,18 +7,18 @@
     "node": ">=12.10.0"
   },
   "devDependencies": {
-    "highlight.js": "10.1.2",
+    "highlight.js": "10.4.1",
     "js-yaml": "3.14.0",
-    "rehype-raw": "4.0.2",
+    "rehype-raw": "5.0.0",
     "rehype-stringify": "8.0.0",
     "remark-gfm": "^1.0.0",
-    "remark-html": "12.0.0",
+    "remark-html": "13.0.1",
```

Comment:
I know these are listed as dev dependencies in the package.json, but they are used to generate our docs so a breaking change especially requires caution. When I do something like this, I make the docs with the packages in master and then make them with the packages in my branch and diff them.

Reply:
(I'm doing this right now and will report what I find.)

Reply:
Quick review shows lots of changes to code block rendering, but a spot check doesn't show anything I can detect visually. Here's the list of changes. If anyone's curious about any of them but doesn't want to build the docs themselves, I can post screenshots. (Building the docs is super easy and fast, though.)

<details>

```
diff -r out/doc/api/all.html out/doc-old/api/all.html
4936c4936
< <h2>Class: assert.AssertionError<a class="srclink" href="https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L165">[src]</a><span><a class="mark" href="#assert_class_assert_assertionerror" id="assert_class_assert_assertionerror">#</a></span></h2>
---
5323c5323
< assert.deepStrictEqual(-<span class="hljs-number">0</span>, -<span class="hljs-number">0</span>);
---
> assert.deepStrictEqual(<span class="hljs-number">-0</span>, <span class="hljs-number">-0</span>);
5327c5327
< assert.deepStrictEqual(<span class="hljs-number">0</span>, -<span class="hljs-number">0</span>);
---
> assert.deepStrictEqual(<span class="hljs-number">0</span>, <span class="hljs-number">-0</span>);
6308,6309c6308,6309
<   <span class="hljs-function"><span class="hljs-title">init</span>(<span class="hljs-params">asyncId, type, triggerAsyncId, resource</span>)</span> { },
<   <span class="hljs-function"><span class="hljs-title">destroy</span>(<span class="hljs-params">asyncId</span>)</span> { }
---
>   init(asyncId, type, triggerAsyncId, resource) { },
>   destroy(asyncId) { }
6313,6314c6313,6314
<   <span class="hljs-function"><span class="hljs-title">init</span>(<span class="hljs-params">asyncId, type, triggerAsyncId, resource</span>)</span> { }
<   <span class="hljs-function"><span class="hljs-title">destroy</span>(<span class="hljs-params">asyncId</span>)</span> {}
---
>   init(asyncId, type, triggerAsyncId, resource) { }
>   destroy(asyncId) {}
6318,6319c6318,6319
<   <span class="hljs-function"><span class="hljs-title">before</span>(<span class="hljs-params">asyncId</span>)</span> { }
<   <span class="hljs-function"><span class="hljs-title">after</span>(<span class="hljs-params">asyncId</span>)</span> { }
---
>   before(asyncId) { }
>   after(asyncId) { }
6426c6426
<   <span class="hljs-function"><span class="hljs-title">init</span>(<span class="hljs-params">asyncId, type, triggerAsyncId</span>)</span> {
---
>   init(asyncId, type, triggerAsyncId) {
6464c6464
<   <span class="hljs-function"><span class="hljs-title">init</span>(<span class="hljs-params">asyncId, type, triggerAsyncId</span>)</span> {
---
>   init(asyncId, type, triggerAsyncId) {
6472c6472
<   <span class="hljs-function"><span class="hljs-title">before</span>(<span class="hljs-params">asyncId</span>)</span> {
---
>   before(asyncId) {
6477c6477
<   <span class="hljs-function"><span class="hljs-title">after</span>(<span class="hljs-params">asyncId</span>)</span> {
---
>   after(asyncId) {
6482c6482
<   <span class="hljs-function"><span class="hljs-title">destroy</span>(<span class="hljs-params">asyncId</span>)</span> {
---
>   destroy(asyncId) {
6622c6622
<   <span class="hljs-function"><span class="hljs-title">init</span>(<span class="hljs-params">asyncId, type, triggerAsyncId, resource</span>)</span> {
---
>   init(asyncId, type, triggerAsyncId, resource) {
6709c6709
< ah.createHook({ <span class="hljs-function"><span class="hljs-title">init</span>(<span class="hljs-params"></span>)</span> {} }).enable(); <span class="hljs-comment">// forces PromiseHooks to be enabled.</span>
---
> ah.createHook({ init() {} }).enable(); <span class="hljs-comment">// forces PromiseHooks to be enabled.</span>
6780c6780
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">db</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(db) {
6785c6785
<   <span class="hljs-function"><span class="hljs-title">getInfo</span>(<span class="hljs-params">query, callback</span>)</span> {
---
>   getInfo(query, callback) {
6791c6791
<   <span class="hljs-function"><span class="hljs-title">close</span>(<span class="hljs-params"></span>)</span> {
---
>   close() {
6870c6870
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">callback</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(callback) {
6875c6875
<   <span class="hljs-function"><span class="hljs-title">done</span>(<span class="hljs-params">err, result</span>)</span> {
---
>   done(err, result) {
6882c6882
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">numThreads</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(numThreads) {
6892c6892
<   <span class="hljs-function"><span class="hljs-title">addNewWorker</span>(<span class="hljs-params"></span>)</span> {
---
>   addNewWorker() {
6920c6920
<   <span class="hljs-function"><span class="hljs-title">runTask</span>(<span class="hljs-params">task, callback</span>)</span> {
---
>   runTask(task, callback) {
6932c6932
<   <span class="hljs-function"><span class="hljs-title">close</span>(<span class="hljs-params"></span>)</span> {
---
>   close() {
7190c7190
< <span class="hljs-keyword">const</span> buf5 = Buffer.from([<span class="hljs-number">257</span>, <span class="hljs-number">257.5</span>, -<span class="hljs-number">255</span>, <span class="hljs-string">'1'</span>]);
---
> <span class="hljs-keyword">const</span> buf5 = Buffer.from([<span class="hljs-number">257</span>, <span class="hljs-number">257.5</span>, <span class="hljs-number">-255</span>, <span class="hljs-string">'1'</span>]);
8201c8201
< <span class="hljs-built_in">console</span>.log(utf16Buffer.indexOf(<span class="hljs-string">'\u03a3'</span>, -<span class="hljs-number">4</span>, <span class="hljs-string">'utf16le'</span>));
---
> <span class="hljs-built_in">console</span>.log(utf16Buffer.indexOf(<span class="hljs-string">'\u03a3'</span>, <span class="hljs-number">-4</span>, <span class="hljs-string">'utf16le'</span>));
8291c8291
< <span class="hljs-built_in">console</span>.log(utf16Buffer.lastIndexOf(<span class="hljs-string">'\u03a3'</span>, -<span class="hljs-number">5</span>, <span class="hljs-string">'utf16le'</span>));
---
> <span class="hljs-built_in">console</span>.log(utf16Buffer.lastIndexOf(<span class="hljs-string">'\u03a3'</span>, <span class="hljs-number">-5</span>, <span class="hljs-string">'utf16le'</span>));
8523c8523
< <pre><code class="language-js"><span class="hljs-keyword">const</span> buf = Buffer.from([-<span class="hljs-number">1</span>, <span class="hljs-number">5</span>]);
---
> <pre><code class="language-js"><span class="hljs-keyword">const</span> buf = Buffer.from([<span class="hljs-number">-1</span>, <span class="hljs-number">5</span>]);
8705c8705
< <pre><code class="language-js"><span class="hljs-keyword">const</span> buf = Buffer.from([<span class="hljs-number">1</span>, -<span class="hljs-number">2</span>]);
---
> <pre><code class="language-js"><span class="hljs-keyword">const</span> buf = Buffer.from([<span class="hljs-number">1</span>, <span class="hljs-number">-2</span>]);
8919c8919
< <span class="hljs-built_in">console</span>.log(buf.subarray(-<span class="hljs-number">6</span>, -<span class="hljs-number">1</span>).toString());
---
> <span class="hljs-built_in">console</span>.log(buf.subarray(<span class="hljs-number">-6</span>, <span class="hljs-number">-1</span>).toString());
8923c8923
< <span class="hljs-built_in">console</span>.log(buf.subarray(-<span class="hljs-number">6</span>, -<span class="hljs-number">2</span>).toString());
---
> <span class="hljs-built_in">console</span>.log(buf.subarray(<span class="hljs-number">-6</span>, <span class="hljs-number">-2</span>).toString());
8927c8927
< <span class="hljs-built_in">console</span>.log(buf.subarray(-<span class="hljs-number">5</span>, -<span class="hljs-number">2</span>).toString());
---
> <span class="hljs-built_in">console</span>.log(buf.subarray(<span class="hljs-number">-5</span>, <span class="hljs-number">-2</span>).toString());
9382c9382
< buf.writeInt8(-<span class="hljs-number">2</span>, <span class="hljs-number">1</span>);
---
> buf.writeInt8(<span class="hljs-number">-2</span>, <span class="hljs-number">1</span>);
11666c11666
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
11744c11744
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
11756c11756
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
12729c12729
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
12743c12743
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
12757c12757
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
12765c12765
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
12785c12785
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
14159c14159
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
28400c28400
< <pre><code class="language-js"><span class="hljs-built_in">require</span>(<span class="hljs-string">'net'</span>).connect(-<span class="hljs-number">1</span>);
---
> <pre><code class="language-js"><span class="hljs-built_in">require</span>(<span class="hljs-string">'net'</span>).connect(<span class="hljs-number">-1</span>);
30653c30653
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params"></span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>() {
30662c30662
<   <span class="hljs-function"><span class="hljs-title">destroy</span>(<span class="hljs-params">err</span>)</span> {
---
>   destroy(err) {
31055c31055
<   <span class="hljs-function"><span class="hljs-title">handleEvent</span>(<span class="hljs-params">event</span>)</span> {
---
>   handleEvent(event) {
31061c31061
<   <span class="hljs-keyword">async</span> <span class="hljs-function"><span class="hljs-title">handleEvent</span>(<span class="hljs-params">event</span>)</span> {
---
>   <span class="hljs-keyword">async</span> handleEvent(event) {
31518c31518
< (<span class="hljs-keyword">async</span> <span class="hljs-function"><span class="hljs-keyword">function</span>(<span class="hljs-params"><span class="hljs-keyword">from</span>, to</span>) </span>{
---
> (<span class="hljs-keyword">async</span> <span class="hljs-function"><span class="hljs-keyword">function</span>(<span class="hljs-params">from, to</span>) </span>{
41906c41906
< is 0. The maximum allowed value is 22<sup>32</sup>-1. A negative value sets
---
> is 0. The maximum allowed value is 2<sup>32</sup>-1. A negative value sets
42171c42171
< is 22<sup>32</sup>-1. <strong>Default:</strong> <code>4096</code>.</li>
---
> is 2<sup>32</sup>-1. <strong>Default:</strong> <code>4096</code>.</li>
42179c42179
< 22<sup>24</sup>-1. <strong>Default:</strong> <code>16384</code>.</li>
---
> 2<sup>24</sup>-1. <strong>Default:</strong> <code>16384</code>.</li>
42188c42188
< maximum allowed value is 22<sup>32</sup>-1. <strong>Default:</strong> <code>65535</code>.</li>
---
> maximum allowed value is 2<sup>32</sup>-1. <strong>Default:</strong> <code>65535</code>.</li>
43916c43916
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">width</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(width) {
43920c43920
<   <span class="hljs-function"><span class="hljs-title">area</span>(<span class="hljs-params"></span>)</span> {
---
>   area() {
44257c44257
< <pre><code class="language-js">(<span class="hljs-function"><span class="hljs-keyword">function</span>(<span class="hljs-params"><span class="hljs-built_in">exports</span>, <span class="hljs-built_in">require</span>, <span class="hljs-built_in">module</span>, __filename, __dirname</span>) </span>{
---
> <pre><code class="language-js">(<span class="hljs-function"><span class="hljs-keyword">function</span>(<span class="hljs-params">exports, require, module, __filename, __dirname</span>) </span>{
44539c44539
< <pre><code class="language-js"><span class="hljs-function"><span class="hljs-keyword">function</span> <span class="hljs-title">require</span>(<span class="hljs-params"><span class="hljs-comment">/* ... */</span></span>) </span>{
---
> <pre><code class="language-js"><span class="hljs-function"><span class="hljs-keyword">function</span> <span class="hljs-title">require</span>(<span class="hljs-params"><span class="hljs-regexp">/* ... */</span></span>) </span>{
50339c50339
<   <span class="hljs-function"><span class="hljs-title">init</span>(<span class="hljs-params">id, type</span>)</span> {
---
>   init(id, type) {
50345c50345
<   <span class="hljs-function"><span class="hljs-title">destroy</span>(<span class="hljs-params">id</span>)</span> {
---
>   destroy(id) {
55290c55290
<   <span class="hljs-keyword">async</span> <span class="hljs-function"><span class="hljs-title">clientHelloHandler</span>(<span class="hljs-params">alpn, servername, ciphers</span>)</span> {
---
>   <span class="hljs-keyword">async</span> clientHelloHandler(alpn, servername, ciphers) {
56403c56403
<   <span class="hljs-function"><span class="hljs-title">action</span>(<span class="hljs-params">name</span>)</span> {
---
>   action(name) {
57197c57197
< <h1>Stream<a class="srclink" href="https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/stream.js#L39">[src]</a><span><a class="mark" href="#stream_stream" id="stream_stream">#</a></span></h1>
---
58760c58760
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">{ highWaterMark, ...options }</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>({ highWaterMark, ...options }) {
58825c58825
<   <span class="hljs-function"><span class="hljs-title">construct</span>(<span class="hljs-params">callback</span>)</span> {
---
>   construct(callback) {
58828c58828
<   <span class="hljs-function"><span class="hljs-title">write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   write(chunk, encoding, callback) {
58831c58831
<   <span class="hljs-function"><span class="hljs-title">destroy</span>(<span class="hljs-params"></span>)</span> {
---
>   destroy() {
58895c58895
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">options</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(options) {
58915c58915
<   <span class="hljs-function"><span class="hljs-title">write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   write(chunk, encoding, callback) {
58918c58918
<   <span class="hljs-function"><span class="hljs-title">writev</span>(<span class="hljs-params">chunks, callback</span>)</span> {
---
>   writev(chunks, callback) {
58941c58941
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">filename</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(filename) {
58946c58946
<   <span class="hljs-function"><span class="hljs-title">_construct</span>(<span class="hljs-params">callback</span>)</span> {
---
>   _construct(callback) {
58956c58956
<   <span class="hljs-function"><span class="hljs-title">_write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   _write(chunk, encoding, callback) {
58959c58959
<   <span class="hljs-function"><span class="hljs-title">_destroy</span>(<span class="hljs-params">err, callback</span>)</span> {
---
>   _destroy(err, callback) {
59070c59070
<   <span class="hljs-function"><span class="hljs-title">write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   write(chunk, encoding, callback) {
59086c59086
<   <span class="hljs-function"><span class="hljs-title">_write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   _write(chunk, encoding, callback) {
59103c59103
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">options</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(options) {
59108c59108
<   <span class="hljs-function"><span class="hljs-title">_write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   _write(chunk, encoding, callback) {
59115c59115
<   <span class="hljs-function"><span class="hljs-title">_final</span>(<span class="hljs-params">callback</span>)</span> {
---
>   _final(callback) {
59173c59173
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">options</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(options) {
59193c59193
<   <span class="hljs-function"><span class="hljs-title">read</span>(<span class="hljs-params">size</span>)</span> {
---
>   read(size) {
59216c59216
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">filename</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(filename) {
59221c59221
<   <span class="hljs-function"><span class="hljs-title">_construct</span>(<span class="hljs-params">callback</span>)</span> {
---
>   _construct(callback) {
59231c59231
<   <span class="hljs-function"><span class="hljs-title">_read</span>(<span class="hljs-params">n</span>)</span> {
---
>   _read(n) {
59241c59241
<   <span class="hljs-function"><span class="hljs-title">_destroy</span>(<span class="hljs-params">err, callback</span>)</span> {
---
>   _destroy(err, callback) {
59329c59329
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">options</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(options) {
59348c59348
<   <span class="hljs-function"><span class="hljs-title">_read</span>(<span class="hljs-params">size</span>)</span> {
---
>   _read(size) {
59365c59365
<   <span class="hljs-function"><span class="hljs-title">read</span>(<span class="hljs-params">size</span>)</span> {
---
>   read(size) {
59381c59381
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">opt</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(opt) {
59387c59387
<   <span class="hljs-function"><span class="hljs-title">_read</span>(<span class="hljs-params"></span>)</span> {
---
>   _read() {
59447c59447
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">options</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(options) {
59466c59466
<   <span class="hljs-function"><span class="hljs-title">read</span>(<span class="hljs-params">size</span>)</span> {
---
>   read(size) {
59469c59469
<   <span class="hljs-function"><span class="hljs-title">write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   write(chunk, encoding, callback) {
59482c59482
<     <span class="hljs-function"><span class="hljs-title">construct</span>(<span class="hljs-params">callback</span>)</span> {
---
>     construct(callback) {
59486c59486
<     <span class="hljs-function"><span class="hljs-title">transform</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>     transform(chunk, encoding, callback) {
59490c59490
<     <span class="hljs-function"><span class="hljs-title">flush</span>(<span class="hljs-params">callback</span>)</span> {
---
>     flush(callback) {
59521c59521
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">source, options</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(source, options) {
59526c59526
<   <span class="hljs-function"><span class="hljs-title">_write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   _write(chunk, encoding, callback) {
59534c59534
<   <span class="hljs-function"><span class="hljs-title">_read</span>(<span class="hljs-params">size</span>)</span> {
---
>   _read(size) {
59557c59557
<   <span class="hljs-function"><span class="hljs-title">transform</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   transform(chunk, encoding, callback) {
59612c59612
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">options</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(options) {
59631c59631
<   <span class="hljs-function"><span class="hljs-title">transform</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   transform(chunk, encoding, callback) {
64529c64529
<   <span class="hljs-function"><span class="hljs-title">write</span>(<span class="hljs-params">data</span>)</span> {
---
>   write(data) {
64863c64863
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">value</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(value) {
64921c64921
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">value</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(value) {
64925c64925
<   <span class="hljs-function"><span class="hljs-title">toString</span>(<span class="hljs-params"></span>)</span> {
---
>   toString() {
65008c65008
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params"></span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>() {
65012c65012
<   <span class="hljs-function"><span class="hljs-title">bar</span>(<span class="hljs-params">callback</span>)</span> {
---
>   bar(callback) {
67406c67406
<       <span class="hljs-function"><span class="hljs-title">initializeImportMeta</span>(<span class="hljs-params">meta</span>)</span> {
---
>       initializeImportMeta(meta) {
diff -r out/doc/api/assert.html out/doc-old/api/assert.html
251c251
< <h2>Class: assert.AssertionError<a class="srclink" href="https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L165">[src]</a><span><a class="mark" href="#assert_class_assert_assertionerror" id="assert_class_assert_assertionerror">#</a></span></h2>
---
638c638
< assert.deepStrictEqual(-<span class="hljs-number">0</span>, -<span class="hljs-number">0</span>);
---
> assert.deepStrictEqual(<span class="hljs-number">-0</span>, <span class="hljs-number">-0</span>);
642c642
< assert.deepStrictEqual(<span class="hljs-number">0</span>, -<span class="hljs-number">0</span>);
---
> assert.deepStrictEqual(<span class="hljs-number">0</span>, <span class="hljs-number">-0</span>);
diff -r out/doc/api/async_hooks.html out/doc-old/api/async_hooks.html
295,296c295,296
<   <span class="hljs-function"><span class="hljs-title">init</span>(<span class="hljs-params">asyncId, type, triggerAsyncId, resource</span>)</span> { },
<   <span class="hljs-function"><span class="hljs-title">destroy</span>(<span class="hljs-params">asyncId</span>)</span> { }
---
>   init(asyncId, type, triggerAsyncId, resource) { },
>   destroy(asyncId) { }
300,301c300,301
<   <span class="hljs-function"><span class="hljs-title">init</span>(<span class="hljs-params">asyncId, type, triggerAsyncId, resource</span>)</span> { }
<   <span class="hljs-function"><span class="hljs-title">destroy</span>(<span class="hljs-params">asyncId</span>)</span> {}
---
>   init(asyncId, type, triggerAsyncId, resource) { }
>   destroy(asyncId) {}
305,306c305,306
<   <span class="hljs-function"><span class="hljs-title">before</span>(<span class="hljs-params">asyncId</span>)</span> { }
<   <span class="hljs-function"><span class="hljs-title">after</span>(<span class="hljs-params">asyncId</span>)</span> { }
---
>   before(asyncId) { }
>   after(asyncId) { }
413c413
<   <span class="hljs-function"><span class="hljs-title">init</span>(<span class="hljs-params">asyncId, type, triggerAsyncId</span>)</span> {
---
>   init(asyncId, type, triggerAsyncId) {
451c451
<   <span class="hljs-function"><span class="hljs-title">init</span>(<span class="hljs-params">asyncId, type, triggerAsyncId</span>)</span> {
---
>   init(asyncId, type, triggerAsyncId) {
459c459
<   <span class="hljs-function"><span class="hljs-title">before</span>(<span class="hljs-params">asyncId</span>)</span> {
---
>   before(asyncId) {
464c464
<   <span class="hljs-function"><span class="hljs-title">after</span>(<span class="hljs-params">asyncId</span>)</span> {
---
>   after(asyncId) {
469c469
<   <span class="hljs-function"><span class="hljs-title">destroy</span>(<span class="hljs-params">asyncId</span>)</span> {
---
>   destroy(asyncId) {
609c609
<   <span class="hljs-function"><span class="hljs-title">init</span>(<span class="hljs-params">asyncId, type, triggerAsyncId, resource</span>)</span> {
---
>   init(asyncId, type, triggerAsyncId, resource) {
696c696
< ah.createHook({ <span class="hljs-function"><span class="hljs-title">init</span>(<span class="hljs-params"></span>)</span> {} }).enable(); <span class="hljs-comment">// forces PromiseHooks to be enabled.</span>
---
> ah.createHook({ init() {} }).enable(); <span class="hljs-comment">// forces PromiseHooks to be enabled.</span>
767c767
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">db</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(db) {
772c772
<   <span class="hljs-function"><span class="hljs-title">getInfo</span>(<span class="hljs-params">query, callback</span>)</span> {
---
>   getInfo(query, callback) {
778c778
<   <span class="hljs-function"><span class="hljs-title">close</span>(<span class="hljs-params"></span>)</span> {
---
>   close() {
857c857
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">callback</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(callback) {
862c862
<   <span class="hljs-function"><span class="hljs-title">done</span>(<span class="hljs-params">err, result</span>)</span> {
---
>   done(err, result) {
869c869
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">numThreads</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(numThreads) {
879c879
<   <span class="hljs-function"><span class="hljs-title">addNewWorker</span>(<span class="hljs-params"></span>)</span> {
---
>   addNewWorker() {
907c907
<   <span class="hljs-function"><span class="hljs-title">runTask</span>(<span class="hljs-params">task, callback</span>)</span> {
---
>   runTask(task, callback) {
919c919
<   <span class="hljs-function"><span class="hljs-title">close</span>(<span class="hljs-params"></span>)</span> {
---
>   close() {
diff -r out/doc/api/buffer.html out/doc-old/api/buffer.html
287c287
< <span class="hljs-keyword">const</span> buf5 = Buffer.from([<span class="hljs-number">257</span>, <span class="hljs-number">257.5</span>, -<span class="hljs-number">255</span>, <span class="hljs-string">'1'</span>]);
---
> <span class="hljs-keyword">const</span> buf5 = Buffer.from([<span class="hljs-number">257</span>, <span class="hljs-number">257.5</span>, <span class="hljs-number">-255</span>, <span class="hljs-string">'1'</span>]);
1298c1298
< <span class="hljs-built_in">console</span>.log(utf16Buffer.indexOf(<span class="hljs-string">'\u03a3'</span>, -<span class="hljs-number">4</span>, <span class="hljs-string">'utf16le'</span>));
---
> <span class="hljs-built_in">console</span>.log(utf16Buffer.indexOf(<span class="hljs-string">'\u03a3'</span>, <span class="hljs-number">-4</span>, <span class="hljs-string">'utf16le'</span>));
1388c1388
< <span class="hljs-built_in">console</span>.log(utf16Buffer.lastIndexOf(<span class="hljs-string">'\u03a3'</span>, -<span class="hljs-number">5</span>, <span class="hljs-string">'utf16le'</span>));
---
> <span class="hljs-built_in">console</span>.log(utf16Buffer.lastIndexOf(<span class="hljs-string">'\u03a3'</span>, <span class="hljs-number">-5</span>, <span class="hljs-string">'utf16le'</span>));
1620c1620
< <pre><code class="language-js"><span class="hljs-keyword">const</span> buf = Buffer.from([-<span class="hljs-number">1</span>, <span class="hljs-number">5</span>]);
---
> <pre><code class="language-js"><span class="hljs-keyword">const</span> buf = Buffer.from([<span class="hljs-number">-1</span>, <span class="hljs-number">5</span>]);
1802c1802
< <pre><code class="language-js"><span class="hljs-keyword">const</span> buf = Buffer.from([<span class="hljs-number">1</span>, -<span class="hljs-number">2</span>]);
---
> <pre><code class="language-js"><span class="hljs-keyword">const</span> buf = Buffer.from([<span class="hljs-number">1</span>, <span class="hljs-number">-2</span>]);
2016c2016
< <span class="hljs-built_in">console</span>.log(buf.subarray(-<span class="hljs-number">6</span>, -<span class="hljs-number">1</span>).toString());
---
> <span class="hljs-built_in">console</span>.log(buf.subarray(<span class="hljs-number">-6</span>, <span class="hljs-number">-1</span>).toString());
2020c2020
< <span class="hljs-built_in">console</span>.log(buf.subarray(-<span class="hljs-number">6</span>, -<span class="hljs-number">2</span>).toString());
---
> <span class="hljs-built_in">console</span>.log(buf.subarray(<span class="hljs-number">-6</span>, <span class="hljs-number">-2</span>).toString());
2024c2024
< <span class="hljs-built_in">console</span>.log(buf.subarray(-<span class="hljs-number">5</span>, -<span class="hljs-number">2</span>).toString());
---
> <span class="hljs-built_in">console</span>.log(buf.subarray(<span class="hljs-number">-5</span>, <span class="hljs-number">-2</span>).toString());
2479c2479
< buf.writeInt8(-<span class="hljs-number">2</span>, <span class="hljs-number">1</span>);
---
> buf.writeInt8(<span class="hljs-number">-2</span>, <span class="hljs-number">1</span>);
diff -r out/doc/api/errors.html out/doc-old/api/errors.html
805c805
< <pre><code class="language-js"><span class="hljs-built_in">require</span>(<span class="hljs-string">'net'</span>).connect(-<span class="hljs-number">1</span>);
---
> <pre><code class="language-js"><span class="hljs-built_in">require</span>(<span class="hljs-string">'net'</span>).connect(<span class="hljs-number">-1</span>);
diff -r out/doc/api/events.html out/doc-old/api/events.html
823c823
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params"></span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>() {
832c832
<   <span class="hljs-function"><span class="hljs-title">destroy</span>(<span class="hljs-params">err</span>)</span> {
---
>   destroy(err) {
1225c1225
<   <span class="hljs-function"><span class="hljs-title">handleEvent</span>(<span class="hljs-params">event</span>)</span> {
---
>   handleEvent(event) {
1231c1231
<   <span class="hljs-keyword">async</span> <span class="hljs-function"><span class="hljs-title">handleEvent</span>(<span class="hljs-params">event</span>)</span> {
---
>   <span class="hljs-keyword">async</span> handleEvent(event) {
diff -r out/doc/api/fs.html out/doc-old/api/fs.html
496c496
< (<span class="hljs-keyword">async</span> <span class="hljs-function"><span class="hljs-keyword">function</span>(<span class="hljs-params"><span class="hljs-keyword">from</span>, to</span>) </span>{
---
> (<span class="hljs-keyword">async</span> <span class="hljs-function"><span class="hljs-keyword">function</span>(<span class="hljs-params">from, to</span>) </span>{
diff -r out/doc/api/http2.html out/doc-old/api/http2.html
2484c2484
< is 0. The maximum allowed value is 22<sup>32</sup>-1. A negative value sets
---
> is 0. The maximum allowed value is 2<sup>32</sup>-1. A negative value sets
2749c2749
< is 22<sup>32</sup>-1. <strong>Default:</strong> <code>4096</code>.</li>
---
> is 2<sup>32</sup>-1. <strong>Default:</strong> <code>4096</code>.</li>
2757c2757
< 22<sup>24</sup>-1. <strong>Default:</strong> <code>16384</code>.</li>
---
> 2<sup>24</sup>-1. <strong>Default:</strong> <code>16384</code>.</li>
2766c2766
< maximum allowed value is 22<sup>32</sup>-1. <strong>Default:</strong> <code>65535</code>.</li>
---
> maximum allowed value is 2<sup>32</sup>-1. <strong>Default:</strong> <code>65535</code>.</li>
diff -r out/doc/api/modules.html out/doc-old/api/modules.html
226c226
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">width</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(width) {
230c230
<   <span class="hljs-function"><span class="hljs-title">area</span>(<span class="hljs-params"></span>)</span> {
---
>   area() {
567c567
< <pre><code class="language-js">(<span class="hljs-function"><span class="hljs-keyword">function</span>(<span class="hljs-params"><span class="hljs-built_in">exports</span>, <span class="hljs-built_in">require</span>, <span class="hljs-built_in">module</span>, __filename, __dirname</span>) </span>{
---
> <pre><code class="language-js">(<span class="hljs-function"><span class="hljs-keyword">function</span>(<span class="hljs-params">exports, require, module, __filename, __dirname</span>) </span>{
849c849
< <pre><code class="language-js"><span class="hljs-function"><span class="hljs-keyword">function</span> <span class="hljs-title">require</span>(<span class="hljs-params"><span class="hljs-comment">/* ... */</span></span>) </span>{
---
> <pre><code class="language-js"><span class="hljs-function"><span class="hljs-keyword">function</span> <span class="hljs-title">require</span>(<span class="hljs-params"><span class="hljs-regexp">/* ... */</span></span>) </span>{
diff -r out/doc/api/n-api.html out/doc-old/api/n-api.html
874c874
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
952c952
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
964c964
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
1937c1937
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
1951c1951
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
1965c1965
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
1973c1973
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
1993c1993
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
3367c3367
< <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-class"><span class="hljs-keyword">enum</span> {</span>
---
> <pre><code class="language-c"><span class="hljs-keyword">typedef</span> <span class="hljs-keyword">enum</span> {
diff -r out/doc/api/perf_hooks.html out/doc-old/api/perf_hooks.html
825c825
<   <span class="hljs-function"><span class="hljs-title">init</span>(<span class="hljs-params">id, type</span>)</span> {
---
>   init(id, type) {
831c831
<   <span class="hljs-function"><span class="hljs-title">destroy</span>(<span class="hljs-params">id</span>)</span> {
---
>   destroy(id) {
diff -r out/doc/api/quic.html out/doc-old/api/quic.html
2581c2581
<   <span class="hljs-keyword">async</span> <span class="hljs-function"><span class="hljs-title">clientHelloHandler</span>(<span class="hljs-params">alpn, servername, ciphers</span>)</span> {
---
>   <span class="hljs-keyword">async</span> clientHelloHandler(alpn, servername, ciphers) {
diff -r out/doc/api/repl.html out/doc-old/api/repl.html
540c540
<   <span class="hljs-function"><span class="hljs-title">action</span>(<span class="hljs-params">name</span>)</span> {
---
>   action(name) {
diff -r out/doc/api/stream.html out/doc-old/api/stream.html
293c293
<         <h1>Stream<a class="srclink" href="https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/stream.js#L39">[src]</a><span><a class="mark" href="#stream_stream" id="stream_stream">#</a></span></h1>
---
1856c1856
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">{ highWaterMark, ...options }</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>({ highWaterMark, ...options }) {
1921c1921
<   <span class="hljs-function"><span class="hljs-title">construct</span>(<span class="hljs-params">callback</span>)</span> {
---
>   construct(callback) {
1924c1924
<   <span class="hljs-function"><span class="hljs-title">write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   write(chunk, encoding, callback) {
1927c1927
<   <span class="hljs-function"><span class="hljs-title">destroy</span>(<span class="hljs-params"></span>)</span> {
---
>   destroy() {
1991c1991
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">options</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(options) {
2011c2011
<   <span class="hljs-function"><span class="hljs-title">write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   write(chunk, encoding, callback) {
2014c2014
<   <span class="hljs-function"><span class="hljs-title">writev</span>(<span class="hljs-params">chunks, callback</span>)</span> {
---
>   writev(chunks, callback) {
2037c2037
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">filename</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(filename) {
2042c2042
<   <span class="hljs-function"><span class="hljs-title">_construct</span>(<span class="hljs-params">callback</span>)</span> {
---
>   _construct(callback) {
2052c2052
<   <span class="hljs-function"><span class="hljs-title">_write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   _write(chunk, encoding, callback) {
2055c2055
<   <span class="hljs-function"><span class="hljs-title">_destroy</span>(<span class="hljs-params">err, callback</span>)</span> {
---
>   _destroy(err, callback) {
2166c2166
<   <span class="hljs-function"><span class="hljs-title">write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   write(chunk, encoding, callback) {
2182c2182
<   <span class="hljs-function"><span class="hljs-title">_write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   _write(chunk, encoding, callback) {
2199c2199
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">options</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(options) {
2204c2204
<   <span class="hljs-function"><span class="hljs-title">_write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   _write(chunk, encoding, callback) {
2211c2211
<   <span class="hljs-function"><span class="hljs-title">_final</span>(<span class="hljs-params">callback</span>)</span> {
---
>   _final(callback) {
2269c2269
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">options</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(options) {
2289c2289
<   <span class="hljs-function"><span class="hljs-title">read</span>(<span class="hljs-params">size</span>)</span> {
---
>   read(size) {
2312c2312
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">filename</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(filename) {
2317c2317
<   <span class="hljs-function"><span class="hljs-title">_construct</span>(<span class="hljs-params">callback</span>)</span> {
---
>   _construct(callback) {
2327c2327
<   <span class="hljs-function"><span class="hljs-title">_read</span>(<span class="hljs-params">n</span>)</span> {
---
>   _read(n) {
2337c2337
<   <span class="hljs-function"><span class="hljs-title">_destroy</span>(<span class="hljs-params">err, callback</span>)</span> {
---
>   _destroy(err, callback) {
2425c2425
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">options</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(options) {
2444c2444
<   <span class="hljs-function"><span class="hljs-title">_read</span>(<span class="hljs-params">size</span>)</span> {
---
>   _read(size) {
2461c2461
<   <span class="hljs-function"><span class="hljs-title">read</span>(<span class="hljs-params">size</span>)</span> {
---
>   read(size) {
2477c2477
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">opt</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(opt) {
2483c2483
<   <span class="hljs-function"><span class="hljs-title">_read</span>(<span class="hljs-params"></span>)</span> {
---
>   _read() {
2543c2543
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">options</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(options) {
2562c2562
<   <span class="hljs-function"><span class="hljs-title">read</span>(<span class="hljs-params">size</span>)</span> {
---
>   read(size) {
2565c2565
<   <span class="hljs-function"><span class="hljs-title">write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   write(chunk, encoding, callback) {
2578c2578
<     <span class="hljs-function"><span class="hljs-title">construct</span>(<span class="hljs-params">callback</span>)</span> {
---
>     construct(callback) {
2582c2582
<     <span class="hljs-function"><span class="hljs-title">transform</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>     transform(chunk, encoding, callback) {
2586c2586
<     <span class="hljs-function"><span class="hljs-title">flush</span>(<span class="hljs-params">callback</span>)</span> {
---
>     flush(callback) {
2617c2617
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">source, options</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(source, options) {
2622c2622
<   <span class="hljs-function"><span class="hljs-title">_write</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   _write(chunk, encoding, callback) {
2630c2630
<   <span class="hljs-function"><span class="hljs-title">_read</span>(<span class="hljs-params">size</span>)</span> {
---
>   _read(size) {
2653c2653
<   <span class="hljs-function"><span class="hljs-title">transform</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   transform(chunk, encoding, callback) {
2708c2708
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">options</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(options) {
2727c2727
<   <span class="hljs-function"><span class="hljs-title">transform</span>(<span class="hljs-params">chunk, encoding, callback</span>)</span> {
---
>   transform(chunk, encoding, callback) {
diff -r out/doc/api/util.html out/doc-old/api/util.html
594c594
<   <span class="hljs-function"><span class="hljs-title">write</span>(<span class="hljs-params">data</span>)</span> {
---
>   write(data) {
928c928
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">value</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(value) {
986c986
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params">value</span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>(value) {
990c990
<   <span class="hljs-function"><span class="hljs-title">toString</span>(<span class="hljs-params"></span>)</span> {
---
>   toString() {
1073c1073
<   <span class="hljs-function"><span class="hljs-title">constructor</span>(<span class="hljs-params"></span>)</span> {
---
>   <span class="hljs-keyword">constructor</span>() {
1077c1077
<   <span class="hljs-function"><span class="hljs-title">bar</span>(<span class="hljs-params">callback</span>)</span> {
---
>   bar(callback) {
diff -r out/doc/api/vm.html out/doc-old/api/vm.html
843c843
<       <span class="hljs-function"><span class="hljs-title">initializeImportMeta</span>(<span class="hljs-params">meta</span>)</span> {
---
>       initializeImportMeta(meta) {
diff -r out/doc/apilinks.json out/doc-old/apilinks.json
2,519c2,519
<   "Agent.defaultMaxSockets": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_agent.js#L198",
<   "agent.createConnection": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/https.js#L97",
<   "agent.getName": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/https.js#L168",
<   "agent.addRequest": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_agent.js#L225",
<   "agent.createSocket": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_agent.js#L296",
<   "agent.removeSocket": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_agent.js#L407",
<   "agent.keepSocketAlive": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_agent.js#L462",
<   "agent.reuseSocket": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_agent.js#L474",
<   "agent.destroy": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_agent.js#L481",
<   "clientrequest._finish": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_client.js#L342",
<   "clientrequest._implicitHeader": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_client.js#L347",
<   "clientrequest.abort": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_client.js#L355",
<   "clientrequest.destroy": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_client.js#L364",
<   "clientrequest.onSocket": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_client.js#L828",
<   "clientrequest._deferToConnect": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_client.js#L846",
<   "clientrequest.setTimeout": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_client.js#L876",
<   "clientrequest.setNoDelay": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_client.js#L904",
<   "clientrequest.setSocketKeepAlive": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_client.js#L908",
<   "clientrequest.clearTimeout": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_client.js#L913",
<   "incomingmessage.setTimeout": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_incoming.js#L142",
<   "incomingmessage._read": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_incoming.js#L150",
<   "incomingmessage.destroy": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_incoming.js#L167",
<   "incomingmessage._addHeaderLines": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_incoming.js#L177",
<   "incomingmessage._addHeaderLine": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_incoming.js#L315",
<   "incomingmessage._dump": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_incoming.js#L342",
<   "outgoingmessage._renderHeaders": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L235",
<   "outgoingmessage.cork": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L255",
<   "outgoingmessage.uncork": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L263",
<   "outgoingmessage.setTimeout": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L271",
<   "outgoingmessage.destroy": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L291",
<   "outgoingmessage._send": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L310",
<   "outgoingmessage._writeRaw": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L335",
<   "outgoingmessage._storeHeader": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L365",
<   "outgoingmessage.setHeader": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L561",
<   "outgoingmessage.getHeader": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L577",
<   "outgoingmessage.getHeaderNames": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L590",
<   "outgoingmessage.getHeaders": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L596",
<   "outgoingmessage.hasHeader": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L613",
<   "outgoingmessage.removeHeader": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L620",
<   "outgoingmessage._implicitHeader": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L650",
<   "outgoingmessage.write": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L671",
<   "outgoingmessage.addTrailers": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L771",
<   "outgoingmessage.end": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L803",
<   "outgoingmessage._finish": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L873",
<   "outgoingmessage._flush": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L898",
<   "outgoingmessage._flushOutput": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L915",
<   "outgoingmessage.flushHeaders": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L944",
<   "outgoingmessage.pipe": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L953",
<   "outgoingmessage[undefined]": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_outgoing.js#L958",
<   "serverresponse._finish": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_server.js#L195",
<   "serverresponse.statusCode": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_server.js#L204",
<   "serverresponse.statusMessage": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_server.js#L205",
<   "serverresponse.assignSocket": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_server.js#L233",
<   "serverresponse.detachSocket": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_server.js#L242",
<   "serverresponse.writeContinue": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_server.js#L249",
<   "serverresponse.writeProcessing": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_server.js#L254",
<   "serverresponse._implicitHeader": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_server.js#L258",
<   "serverresponse.writeHead": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_server.js#L263",
<   "serverresponse.writeHeader": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_http_server.js#L347",
<   "server.setTimeout": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/https.js#L88",
<   "server[undefined]": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L1635",
<   "_tls_common.createSecureContext": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_tls_common.js#L159",
<   "_tls_common.translatePeerCertificate": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_tls_common.js#L383",
<   "_tls_common.SecureContext": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_tls_common.js#L73",
<   "_tls_wrap.createServer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_tls_wrap.js#L1233",
<   "_tls_wrap.connect": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_tls_wrap.js#L1567",
<   "_tls_wrap.TLSSocket": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_tls_wrap.js#L455",
<   "_tls_wrap.Server": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/_tls_wrap.js#L1165",
<   "assert": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L106",
<   "assert.fail": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L122",
<   "assert.AssertionError": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L165",
<   "assert.ok": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L405",
<   "assert.equal": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L412",
<   "assert.notEqual": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L430",
<   "assert.deepEqual": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L447",
<   "assert.notDeepEqual": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L464",
<   "assert.deepStrictEqual": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L481",
<   "assert.notDeepStrictEqual": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L498",
<   "assert.strictEqual": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L514",
<   "assert.notStrictEqual": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L529",
<   "assert.throws": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L839",
<   "assert.rejects": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L843",
<   "assert.doesNotThrow": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L847",
<   "assert.doesNotReject": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L851",
<   "assert.ifError": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L855",
<   "assert.match": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L939",
<   "assert.doesNotMatch": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L943",
<   "assert.CallTracker": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L947",
<   "assert.strict": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/assert.js#L953",
<   "async_hooks.createHook": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L144",
<   "AsyncResource": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L153",
<   "new AsyncResource": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L154",
<   "asyncResource.runInAsyncScope": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L191",
<   "asyncResource.emitDestroy": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L207",
<   "asyncResource.asyncId": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L215",
<   "asyncResource.triggerAsyncId": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L219",
<   "asyncResource.bind": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L244",
<   "AsyncLocalStorage": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L262",
<   "new AsyncLocalStorage": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L263",
<   "asyncLocalStorage.disable": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L268",
<   "asyncLocalStorage._enable": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L280",
<   "asyncLocalStorage._propagate": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L289",
<   "asyncLocalStorage.enterWith": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L296",
<   "asyncLocalStorage.run": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L302",
<   "asyncLocalStorage.exit": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L317",
<   "asyncLocalStorage.getStore": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/async_hooks.js#L329",
<   "Buffer.prototype": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L125",
<   "Buffer.poolSize": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L141",
<   "buffer.Buffer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L270",
<   "Buffer.from": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L296",
<   "Buffer.of": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L342",
<   "Buffer.alloc": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L362",
<   "Buffer.allocUnsafe": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L375",
<   "Buffer.allocUnsafeSlow": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L385",
<   "buffer.SlowBuffer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L392",
<   "Buffer.isBuffer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L512",
<   "Buffer.compare": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L516",
<   "Buffer.isEncoding": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L532",
<   "Buffer[kIsEncodingSymbol]": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L536",
<   "Buffer.concat": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L538",
<   "Buffer.byteLength": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L714",
<   "buf.copy": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L759",
<   "buf.toString": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L768",
<   "buf.equals": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L800",
<   "buf[customInspectSymbol]": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L817",
<   "buf.inspect": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L849",
<   "buf.compare": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L851",
<   "buf.indexOf": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L945",
<   "buf.lastIndexOf": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L949",
<   "buf.includes": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L953",
<   "buf.fill": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L961",
<   "buf.write": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L1036",
<   "buf.toJSON": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L1074",
<   "buf.slice": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L1101",
<   "buf.swap16": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L1115",
<   "buf.swap32": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L1130",
<   "buf.swap64": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L1147",
<   "buf.toLocaleString": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/buffer.js#L1166",
<   "child_process.fork": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/child_process.js#L89",
<   "child_process._forkChild": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/child_process.js#L146",
<   "child_process.exec": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/child_process.js#L178",
<   "child_process.execFile": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/child_process.js#L213",
<   "child_process.spawn": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/child_process.js#L612",
<   "child_process.spawnSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/child_process.js#L622",
<   "child_process.execFileSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/child_process.js#L689",
<   "child_process.execSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/child_process.js#L708",
<   "crypto.createHash": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/crypto.js#L122",
<   "crypto.createCipheriv": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/crypto.js#L130",
<   "crypto.createDecipheriv": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/crypto.js#L138",
<   "crypto.createDiffieHellman": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/crypto.js#L142",
<   "crypto.createDiffieHellmanGroup": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/crypto.js#L146",
<   "crypto.createECDH": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/crypto.js#L150",
<   "crypto.createHmac": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/crypto.js#L154",
<   "crypto.createSign": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/crypto.js#L158",
<   "crypto.createVerify": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/crypto.js#L162",
<   "dgram.Socket": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L93",
<   "dgram.createSocket": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L133",
<   "socket.bind": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L208",
<   "socket.connect": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L911",
<   "socket.disconnect": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L431",
<   "socket.sendto": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L445",
<   "socket.send": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L557",
<   "socket.close": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L713",
<   "socket.address": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L511",
<   "socket.remoteAddress": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L755",
<   "socket.setBroadcast": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L771",
<   "socket.setTTL": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L779",
<   "socket.setMulticastTTL": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L791",
<   "socket.setMulticastLoopback": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L803",
<   "socket.setMulticastInterface": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L813",
<   "socket.addMembership": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L823",
<   "socket.dropMembership": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L839",
<   "socket.addSourceSpecificMembership": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L854",
<   "socket.dropSourceSpecificMembership": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L877",
<   "socket.ref": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L1067",
<   "socket.unref": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L1081",
<   "socket.setRecvBufferSize": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L949",
<   "socket.setSendBufferSize": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L954",
<   "socket.getRecvBufferSize": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L959",
<   "socket.getSendBufferSize": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L964",
<   "socket._healthCheck": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L1020",
<   "socket._stopReceiving": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dgram.js#L1025",
<   "Channel": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/diagnostics_channel.js#L65",
<   "new Channel": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/diagnostics_channel.js#L66",
<   "channel.SymbolHasInstance": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/diagnostics_channel.js#L71",
<   "channel.subscribe": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/diagnostics_channel.js#L77",
<   "channel.publish": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/diagnostics_channel.js#L87",
<   "diagnostics_channel.channel": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/diagnostics_channel.js#L92",
<   "diagnostics_channel.hasSubscribers": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/diagnostics_channel.js#L107",
<   "dns.lookup": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L93",
<   "dns.lookupService": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L169",
<   "resolver.resolveAny": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L236",
<   "resolver.resolve4": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L237",
<   "resolver.resolve6": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L238",
<   "resolver.resolveCaa": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L239",
<   "resolver.resolveCname": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L240",
<   "resolver.resolveMx": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L241",
<   "resolver.resolveNs": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L242",
<   "resolver.resolveTxt": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L243",
<   "resolver.resolveSrv": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L244",
<   "resolver.resolvePtr": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L245",
<   "resolver.resolveNaptr": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L246",
<   "resolver.resolveSoa": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L247",
<   "resolver.reverse": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L248",
<   "resolver.resolve": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L252",
<   "dns.defaultResolverSetServers": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/dns.js#L269",
<   "events.EventEmitter": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/events.js#L83",
<   "events.getEventListeners": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/events.js#L686",
<   "events.once": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/events.js#L708",
<   "events.on": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/events.js#L779",
<   "fs.access": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L215",
<   "fs.accessSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L230",
<   "fs.exists": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L239",
<   "fs.existsSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L265",
<   "fs.readFile": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L326",
<   "fs.readFileSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L393",
<   "fs.close": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L444",
<   "fs.closeSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L453",
<   "fs.open": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L461",
<   "fs.openSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L486",
<   "fs.read": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L503",
<   "fs.readSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L574",
<   "fs.readv": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L615",
<   "fs.readvSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L636",
<   "fs.write": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L654",
<   "fs.writeSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L705",
<   "fs.writev": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L736",
<   "fs.writevSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L759",
<   "fs.rename": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L774",
<   "fs.renameSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L785",
<   "fs.truncate": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L794",
<   "fs.truncateSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L820",
<   "fs.ftruncate": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L841",
<   "fs.ftruncateSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L856",
<   "fs.rmdir": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L871",
<   "fs.rmdirSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L901",
<   "fs.rm": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L916",
<   "fs.rmSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L931",
<   "fs.fdatasync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L938",
<   "fs.fdatasyncSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L945",
<   "fs.fsync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L952",
<   "fs.fsyncSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L959",
<   "fs.mkdir": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L966",
<   "fs.mkdirSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L991",
<   "fs.readdir": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1016",
<   "fs.readdirSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1037",
<   "fs.fstat": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1048",
<   "fs.lstat": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1061",
<   "fs.stat": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1074",
<   "fs.fstatSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1100",
<   "fs.lstatSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1108",
<   "fs.statSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1120",
<   "fs.readlink": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1132",
<   "fs.readlinkSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1141",
<   "fs.symlink": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1151",
<   "fs.symlinkSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1193",
<   "fs.link": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1214",
<   "fs.linkSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1228",
<   "fs.unlink": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1240",
<   "fs.unlinkSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1248",
<   "fs.fchmod": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1255",
<   "fs.fchmodSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1265",
<   "fs.chmod": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1305",
<   "fs.chmodSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1315",
<   "fs.lchown": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1324",
<   "fs.lchownSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1334",
<   "fs.fchown": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1343",
<   "fs.fchownSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1354",
<   "fs.chown": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1364",
<   "fs.chownSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1375",
<   "fs.utimes": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1384",
<   "fs.utimesSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1396",
<   "fs.futimes": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1405",
<   "fs.futimesSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1416",
<   "fs.lutimes": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1425",
<   "fs.lutimesSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1437",
<   "fs.writeFile": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1476",
<   "fs.writeFileSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1508",
<   "fs.appendFile": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1534",
<   "fs.appendFileSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1548",
<   "fs.watch": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1561",
<   "fs.watchFile": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1592",
<   "fs.unwatchFile": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1632",
<   "fs.realpathSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1704",
<   "fs.realpath": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1846",
<   "fs.mkdtemp": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1983",
<   "fs.mkdtempSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L1997",
<   "fs.copyFile": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L2013",
<   "fs.copyFileSync": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L2035",
<   "fs.createReadStream": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L2055",
<   "fs.createWriteStream": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/fs.js#L2060",
<   "http.createServer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/http.js#L45",
<   "http.request": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/http.js#L49",
<   "http.get": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/http.js#L53",
<   "https.Server": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/https.js#L47",
<   "https.createServer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/https.js#L90",
<   "https.Agent": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/https.js#L148",
<   "agent._getSession": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/https.js#L242",
<   "agent._cacheSession": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/https.js#L246",
<   "agent._evictSession": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/https.js#L268",
<   "https.request": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/https.js#L280",
<   "https.get": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/https.js#L316",
<   "Session": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/inspector.js#L46",
<   "new Session": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/inspector.js#L47",
<   "session.connect": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/inspector.js#L54",
<   "session.connectToMainThread": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/inspector.js#L61",
<   "session.onMessageSymbol": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/inspector.js#L71",
<   "session.post": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/inspector.js#L94",
<   "session.disconnect": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/inspector.js#L121",
<   "inspector.inspectorOpen": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/inspector.js#L135",
<   "inspector.inspectorWaitForDebugger": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/inspector.js#L144",
<   "net.createServer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L165",
<   "net.connect": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L182",
<   "net.normalizeArgs": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L206",
<   "net.Socket": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L272",
<   "socket._unrefTimer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L391",
<   "socket._final": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L401",
<   "socket.setTimeout": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L459",
<   "socket._onTimeout": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L462",
<   "socket.setNoDelay": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L480",
<   "socket.setKeepAlive": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L498",
<   "socket._read": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L571",
<   "socket.end": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L583",
<   "socket.pause": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L590",
<   "socket.resume": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L604",
<   "socket.read": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L613",
<   "socket.destroySoon": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L630",
<   "socket._destroy": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L641",
<   "socket._getpeername": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L681",
<   "socket._getsockname": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L717",
<   "socket[kAfterAsyncWrite]": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L739",
<   "socket._writeGeneric": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L743",
<   "socket._writev": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L775",
<   "socket._write": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L780",
<   "net.Server": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L1147",
<   "net.createServerHandle": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L1185",
<   "server._listen2": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L1314",
<   "server.listen": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L1370",
<   "server.address": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L1489",
<   "server.getConnections": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L1536",
<   "server.close": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L1574",
<   "server._emitCloseIfDrained": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L1613",
<   "server._setupWorker": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L1660",
<   "server.ref": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L1669",
<   "server.unref": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/net.js#L1678",
<   "os.loadavg": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/os.js#L100",
<   "os.cpus": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/os.js#L105",
<   "os.arch": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/os.js#L126",
<   "os.platform": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/os.js#L131",
<   "os.tmpdir": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/os.js#L136",
<   "os.endianness": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/os.js#L158",
<   "os.networkInterfaces": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/os.js#L207",
<   "os.setPriority": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/os.js#L237",
<   "os.getPriority": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/os.js#L252",
<   "os.userInfo": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/os.js#L267",
<   "PerformanceObserver": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/perf_hooks.js#L343",
<   "new PerformanceObserver": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/perf_hooks.js#L344",
<   "performanceObserver.disconnect": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/perf_hooks.js#L378",
<   "performanceObserver.observe": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/perf_hooks.js#L395",
<   "perf_hooks.monitorEventLoopDelay": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/perf_hooks.js#L653",
<   "QueryString": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/querystring.js#L45",
<   "readline.createInterface": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L115",
<   "readline.Interface": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L120",
<   "interface.setPrompt": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L318",
<   "interface.getPrompt": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L323",
<   "interface._setRawMode": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L328",
<   "interface.prompt": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L339",
<   "interface.question": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L350",
<   "interface._onLine": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L364",
<   "interface._writeToOutput": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L375",
<   "interface._addHistory": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L383",
<   "interface._refreshLine": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L410",
<   "interface.close": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L451",
<   "interface.pause": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L462",
<   "interface.resume": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L471",
<   "interface.write": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L480",
<   "interface._normalWrite": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L489",
<   "interface._insertString": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L523",
<   "interface._tabComplete": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L542",
<   "interface._wordLeft": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L608",
<   "interface._wordRight": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L621",
<   "interface._deleteLeft": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L629",
<   "interface._deleteRight": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L642",
<   "interface._deleteWordLeft": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L653",
<   "interface._deleteWordRight": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L671",
<   "interface._deleteLineLeft": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L682",
<   "interface._deleteLineRight": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L689",
<   "interface.clearLine": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L695",
<   "interface._line": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L704",
<   "interface._historyNext": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L717",
<   "interface._historyPrev": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L737",
<   "interface._getDisplayPos": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L758",
<   "interface.getCursorPos": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L791",
<   "interface._getCursorPos": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L796",
<   "interface._moveCursor": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L800",
<   "interface._ttyWrite": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L874",
<   "interface[SymbolAsyncIterator]": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L1119",
<   "readline.emitKeypressEvents": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L1161",
<   "readline.cursorTo": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L1230",
<   "readline.moveCursor": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L1256",
<   "readline.clearLine": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L1290",
<   "readline.clearScreenDown": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/readline.js#L1312",
<   "repl.REPLServer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/repl.js#L209",
<   "repl.start": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/repl.js#L970",
<   "replserver.setupHistory": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/repl.js#L975",
<   "replserver.clearBufferedCommand": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/repl.js#L979",
<   "replserver.close": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/repl.js#L983",
<   "replserver.createContext": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/repl.js#L997",
<   "replserver.resetContext": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/repl.js#L1047",
<   "replserver.displayPrompt": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/repl.js#L1084",
<   "replserver.setPrompt": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/repl.js#L1099",
<   "replserver.complete": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/repl.js#L1149",
<   "replserver.completeOnEditorMode": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/repl.js#L1416",
<   "replserver.defineCommand": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/repl.js#L1429",
<   "repl.Recoverable": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/repl.js#L1630",
<   "Stream": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/stream.js#L39",
<   "Stream.Readable": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/stream.js#L40",
<   "Stream.Writable": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/stream.js#L41",
<   "Stream.Duplex": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/stream.js#L42",
<   "Stream.Transform": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/stream.js#L43",
<   "Stream.PassThrough": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/stream.js#L44",
<   "Stream.pipeline": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/stream.js#L45",
<   "Stream.finished": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/stream.js#L46",
<   "Stream.Stream": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/stream.js#L76",
<   "Stream._isUint8Array": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/stream.js#L78",
<   "Stream._uint8ArrayToBuffer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/stream.js#L79",
<   "string_decoder.StringDecoder": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/string_decoder.js#L70",
<   "timers.setTimeout": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/timers.js#L134",
<   "timers.clearTimeout": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/timers.js#L173",
<   "timers.setInterval": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/timers.js#L188",
<   "timers.clearInterval": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/timers.js#L218",
<   "timers.setImmediate": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/timers.js#L239",
<   "timers.clearImmediate": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/timers.js#L275",
<   "tls.convertALPNProtocols": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tls.js#L140",
<   "tls.checkServerIdentity": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tls.js#L232",
<   "trace_events.createTracing": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/trace_events.js#L81",
<   "tty.isatty": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tty.js#L42",
<   "tty.ReadStream": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tty.js#L46",
<   "readstream.setRawMode": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tty.js#L73",
<   "tty.WriteStream": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tty.js#L84",
<   "writestream.isTTY": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tty.js#L120",
<   "writestream.getColorDepth": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tty.js#L122",
<   "writestream.hasColors": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tty.js#L124",
<   "writestream._refreshSize": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tty.js#L126",
<   "writestream.cursorTo": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tty.js#L144",
<   "writestream.moveCursor": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tty.js#L148",
<   "writestream.clearLine": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tty.js#L152",
<   "writestream.clearScreenDown": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tty.js#L156",
<   "writestream.getWindowSize": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/tty.js#L160",
<   "url.Url": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/url.js#L56",
<   "url.urlParse": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/url.js#L151",
<   "url.parse": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/url.js#L159",
<   "url.urlFormat": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/url.js#L540",
<   "url.format": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/url.js#L576",
<   "url.urlResolve": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/url.js#L663",
<   "url.resolve": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/url.js#L667",
<   "url.urlResolveObject": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/url.js#L671",
<   "url.resolveObject": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/url.js#L676",
<   "url.parseHost": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/url.js#L953",
<   "util.isBoolean": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L67",
<   "util.isNull": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L71",
<   "util.isNullOrUndefined": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L75",
<   "util.isNumber": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L79",
<   "util.isString": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L83",
<   "util.isSymbol": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L87",
<   "util.isUndefined": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L91",
<   "util.isObject": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L95",
<   "util.isError": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L99",
<   "util.isFunction": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L103",
<   "util.isPrimitive": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L107",
<   "util.log": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L130",
<   "util.inherits": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L152",
<   "util._extend": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L172",
<   "util.callbackify": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L195",
<   "util.getSystemErrorName": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/util.js#L229",
<   "v8.writeHeapSnapshot": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L59",
<   "v8.getHeapSnapshot": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L67",
<   "Serializer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L76",
<   "Deserializer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L78",
<   "v8.setFlagsFromString": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L118",
<   "v8.getHeapStatistics": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L123",
<   "v8.getHeapSpaceStatistics": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L143",
<   "v8.getHeapCodeStatistics": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L161",
<   "serializer._getDataCloneError": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L175",
<   "deserializer.readRawBytes": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L177",
<   "DefaultSerializer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L209",
<   "new DefaultSerializer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L210",
<   "defaultSerializer._writeHostObject": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L216",
<   "DefaultDeserializer": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L237",
<   "defaultDeserializer._readHostObject": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L238",
<   "v8.serialize": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L260",
<   "v8.deserialize": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/v8.js#L267",
<   "Script": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/vm.js#L63",
<   "new Script": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/vm.js#L64",
<   "script.runInThisContext": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/vm.js#L128",
<   "script.runInContext": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/vm.js#L136",
<   "script.runInNewContext": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/vm.js#L146",
<   "vm.isContext": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/vm.js#L212",
<   "vm.createContext": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/vm.js#L220",
<   "vm.createScript": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/vm.js#L261",
<   "vm.runInContext": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/vm.js#L283",
<   "vm.runInNewContext": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/vm.js#L297",
<   "vm.runInThisContext": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/vm.js#L306",
<   "vm.compileFunction": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/vm.js#L313",
<   "vm.measureMemory": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/vm.js#L389",
<   "WASI": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/wasi.js#L54",
<   "new WASI": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/wasi.js#L55",
<   "wASI.start": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/wasi.js#L106",
<   "wASI.initialize": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/wasi.js#L137",
<   "zlib.Deflate": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/zlib.js#L716",
<   "zlib.Inflate": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/zlib.js#L724",
<   "zlib.Gzip": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/zlib.js#L732",
<   "zlib.Gunzip": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/zlib.js#L740",
<   "zlib.DeflateRaw": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/zlib.js#L748",
<   "zlib.InflateRaw": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/zlib.js#L757",
<   "zlib.Unzip": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/zlib.js#L765",
<   "zlib.BrotliCompress": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/zlib.js#L839",
<   "zlib.BrotliDecompress": "https://github.com/nodejs/node/blob/306c15c15abab524769b43bd0b1b3f995c259e66/lib/zlib.js#L847"
---
```

</details>

Reply:
Sample improvement: The `assert` docs now highlight `-0` as a number instead of just `0` and leaving them `-` un-highlighted:

Before:

![image](https://user-images.githubusercontent.com/718899/101372477-ccdd4080-3860-11eb-9041-15f72f6ad66c.png)

After:

![image](https://user-images.githubusercontent.com/718899/101372502-d4044e80-3860-11eb-848c-ac5a0b1fb196.png)

Reply:
Sample "I don't see any difference despite the underlying code changing": The `vm` module has this line changed but I don't see any difference.

Before:

![image](https://user-images.githubusercontent.com/718899/101372663-ff873900-3860-11eb-94cd-f8b35783f4bf.png)


After:

![image](https://user-images.githubusercontent.com/718899/101372733-0f9f1880-3861-11eb-8cee-ccd062f23ccf.png)